cli/chaosli/cmd/create.go

```diff
@@ -674,6 +678,36 @@ For example, if you set both "app=hello-node" and "pod-name=ubuntu-uuid", then n
 	return selectorLabels
 }
 
+func getFilters() *v1beta1.DisruptionFilter {
+	a := confirmOption("Would you like to filter targets based on annotations", "We will remove any targets that don't contain matching annotations")
+	if a {
+		filters := getSliceInput(
+			"Add an annotation for filtering.",
+			`Please specify this in the form of "key=value", e.g., "app=hello-node". One annotation per new-line. If you specify multiple, we will only target the union of all.
+For example, if you set both "app=hello-node" and "pod-name=ubuntu-uuid", then no matter how many pods with the annotation "app=hello-node" there were, the disruption would only target pods who also had the annotation "pod-name=ubuntu-uuid".`,
+			survey.WithValidator(survey.Required),
+			survey.WithValidator(getLabelsSetValidator()),
+		)
+
+		var filterLabels labels.Set
+
+		for _, f := range filters {
```

Comment:
can we merge (union) all the filters first, then call `labels`.Merge with the unified filters? Because `filterLabels` are already a `Set`, where `labels` is a customized DS

Reply:
Asked in slack, but I'm not sure how that would change the code, could you write a snippet?

Reply:
yup, I see what you meant here 🙇